src/components/editor/Editor.jsx

```diff
@@ -3,7 +3,9 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
-import { removeAllItems, assignBaseURL, runValidation } from '../../actions/index'
+import {
+  removeAllItems, assignBaseURL, runValidation, showGroupChooser,
+} from '../../actions/index'
```

Comment:
if it's getting too big for a single line ... should we bother to import the pieces, or just import the whole thing?

Reply:
i've always had a slight personal preference for explicit and limited imports, but it's not all that strong